content/js/users.md

```diff
@@ -210,10 +197,35 @@ skygear.publicDB.query(query).then((records) => {
 });
 ```
 
+## Others
```

Comment:
Others may not a good name. And all these h2 should be present in the submenu. How about we simply remove "User relations" and "User access control", and make the "Other user lookup" a submenu item?